src/dom_entity_names/events.rs

```diff
@@ -4,7 +4,7 @@ macro_rules! make_events {
     { $($event_camel:ident => $event:expr),+ } => {
 
         /// The Ev enum restricts element-creation to only valid event names, as defined here:
-        /// [https://developer.mozilla.org/en-US/docs/Web/Evs](https://developer.mozilla.org/en-US/docs/Web/Evs)
+        /// [https://developer.mozilla.org/en-US/docs/Web/Evs](https://developer.mozilla.org/en-US/docs/Web/Events)
```

Comment:
Please, change also the link description to `[MDN reference]`, thanks

Reply:
It should also fix linter error.
Run `cargo make verify` to make sure it's ok before pushing.

Reply:
@MartinKavik would `[MDN reference Web/Events]` also be okay?

Reply:
yes

Reply:
@MartinKavik pushed